### PR TITLE
New version: Streamlink.Streamlink version 7.0.0-1

### PR DESCRIPTION
**Pull request by fancybody**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/188906)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:


(Automated response - build 897.)

File: manifests/s/Streamlink/Streamlink/7.0.0-1/Streamlink.Streamlink.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  InstallerUrl: https://github.com/streamlink/windows-builds/releases/download/7.0.0-1/streamlink-7.0.0-1-py312-x86_64.exe
+  InstallerSha256: 2D1EBC7385D4974EF36D89406491E82AB416150130CAD96105C9733E2FE837F3
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x86
+  Scope: machine
+  InstallerUrl: https://github.com/streamlink/windows-builds/releases/download/7.0.0-1/streamlink-7.0.0-1-py312-x86_64.exe
+  InstallerSha256: 2D1EBC7385D4974EF36D89406491E82AB416150130CAD96105C9733E2FE837F3
+  InstallerSwitches:
+    Custom: /ALLUSERS
```
Comment (bastimeyer) [inline manifests/s/Streamlink/Streamlink/7.0.0-1/Streamlink.Streamlink.installer.yaml:25]: That's not correct. `x86_64` is not `x86`.

File: manifests/s/Streamlink/Streamlink/7.0.0-1/Streamlink.Streamlink.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+Publisher: Streamlink
+PublisherUrl: https://github.com/streamlink/streamlink
+PublisherSupportUrl: https://github.com/streamlink/streamlink/issues
+Author: Streamlink
+PackageName: Streamlink
+PackageUrl: https://streamlink.github.io/
+License: BSD-2-Clause
+LicenseUrl: https://github.com/streamlink/windows-builds/blob/HEAD/LICENSE
+Copyright: Copyright (c) 2016-2022, Streamlink Team
+CopyrightUrl: https://raw.githubusercontent.com/streamlink/streamlink/master/LICENSE
```
Comment (bastimeyer) [inline manifests/s/Streamlink/Streamlink/7.0.0-1/Streamlink.Streamlink.locale.en-US.yaml:16]: Incorrect copyright date and license data. The license data of the Streamlink project should be used, not the one of the Windows builds repo, which is just a bunch of BASH scripts for building installers and portable archives.

File: manifests/s/Streamlink/Streamlink/7.0.0-1/Streamlink.Streamlink.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  Streamlink 7.0.0 changes:
+  - ⚠️ Support for Python 3.8 and Windows 7/8 systems has ended.
+  - ⚠️ No more x86 ("32 bit") builds.
+  Please note that the x86_64-installer's default location differs from x86.
+  Users of non-x86_64 systems can always build Streamlink+dependencies on their own.
+
+  ⚙️ Instructions
+  See the README.md for all the details about the installers and portable archives.Further information can be found in Streamlink's install docs and Command-Line Interface usage guide.❤️ Support
+  If you think that Streamlink is useful and if you want to keep the project alive, then please consider supporting its maintainers by sending a small and optionally recurring tip via the available options.
+  Your support is very much appreciated, thank you!
```
Comment (bastimeyer) [inline manifests/s/Streamlink/Streamlink/7.0.0-1/Streamlink.Streamlink.locale.en-US.yaml:39]: This is misformatted and also doesn't reflect the actual Streamlink changelog.

**Review by stephengillie — APPROVED** [review]

**Review by bastimeyer — APPROVED** [review]

**Comment by bastimeyer** [comment]
> Unresolved conversations
> 3 conversations must be resolved before merging.

I can't resolve the comments of my previous review, btw...

**Comment by bastimeyer** [comment]
Since I don't want to continue being spammed by the bot, I deleted my review comments, because as said, I am not able to resolve the comments. I don't have the permissions here. With the comments deleted however, there are no merge restrictions anymore.

**Pull request merged**